buildkit/extraction.py

```diff
@@ -67,6 +67,8 @@ def _process_relative_to(unpack_root, relative_to):
         raise BuildkitAbort()
     for src_path in relative_root.iterdir():
         dest_path = unpack_root / src_path.name
+        if os.path.exists(dest_path):
+            os.remove(dest_path)
```

Comment:
On Windows `rename` does not work if the file already exists

Reply:
you mean remove i guess

Reply:
It is the user's responsibility to clean up the source tree after a failed attempt. Removing files preemptively is dangerous if the user makes a mistake with their paths. At most, I would tell the user what happen and recommend a course of action.

(Also FYI, this code breaks under Python 3.5 because it doesn't implement PEP 519)

Reply:
@intika I was talking about [this line](https://github.com/Eloston/ungoogled-chromium/blob/e12fe24306d951f27f0750c115b34cfb12ef313e/buildkit/extraction.py#L72) in fact